src/components/ProgressBar.ts

```diff
@@ -75,12 +79,24 @@ class ProgressBar extends Component<ProgressBarProps, {}> {
         return Math.round((progress / maximumValue) * 100);
     }
 
-    private getProgressText(maximumValue: number, progress?: number): string {
-        if (progress || progress === 0) {
-            return maximumValue < 1 ? "Invalid" : `${this.calculatePercentage(progress, maximumValue)}%`;
+    private setProgress(maximumValue: number, value: number | undefined, text?: DisplayText, displayTextValue?: string): string {
```

Comment:
function name does not match wat it is doing... `set<Some>` implies that nothing is returned...

Reply:
Renamed